src/lib/modules/blockchain_connector/index.js

```diff
@@ -662,7 +651,7 @@ class BlockchainConnector {
       s: `0x${tx.s.toString('hex').toLowerCase()}`,
       value: tx.value.toString('utf8'),
       to: `0x${tx.to.toString('hex').toLowerCase()}`,
-      rawHash: hash
+      hash
```

Comment:
We still need to attach the `hash` itself here because we use it to find the transaction in the client's store.

Now this part can be confusing because users will see `hash` and might thing it's the transaction hash, but actually it's the `raw` transaction hash. I can rename this to `rawHash` again to make it clear.